common/model/tests/test_transaction.c

```diff
@@ -49,12 +49,125 @@ void test_reset(void) {
   transaction_free(transaction);
 }
 
+void test_read_write_transaction_obj(void) {
+  flex_trit_t test_address[FLEX_TRIT_SIZE_243];
+  int64_t test_value = 123456;
+  flex_trit_t test_obsolete_tag[FLEX_TRIT_SIZE_81];
+  uint64_t test_timestamp = 1477037811737;
+  int64_t test_current_index = 2;
+  int64_t test_last_index = 5;
+  flex_trit_t test_bundle[FLEX_TRIT_SIZE_243];
+
+  flex_trit_t test_trunk[FLEX_TRIT_SIZE_243];
+  flex_trit_t test_branch[FLEX_TRIT_SIZE_243];
+  uint64_t test_attachment_timestamp = 1477037821737;
+  uint64_t test_attachment_timestamp_lower = 1477037831737;
+  uint64_t test_attachment_timestamp_upper = 1477037841737;
```

Comment:
Can you also put all these test values in `common/model/tests/defs.h` ? They could be useful elsewhere.